WordPress/src/main/java/org/wordpress/android/modules/WordPressGlideModule.kt

```diff
@@ -24,7 +24,7 @@ import javax.inject.Named
 @GlideModule
 class WordPressGlideModule : AppGlideModule() {
     @Inject
-    @Named("custom-ssl")
+    @Named("custom-ssl-custom-redirects")
     lateinit var requestQueue: RequestQueue
```

Comment:
I found the `requestQueue` is a param of `VolleyUrlLoader.Factory(requestQueue, glideRequestFactory)`.
And I found https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/AppInitializer.kt#L676 which is also related to Volley. Should we replace the `requestQueue` here with the `@Named("custom-ssl-custom-redirects")` one?

Reply:
Good question @jarvislin 👍 
My understanding is the `AppInitializer` `requestQueue` queue handles the api calls and we don't need to implement the custom redirection there. For now we only need the custom redirection for images which are handled by the `GlideModule`.

Reply:
Got it! Thanks for the explanation 👍